climpred/comparisons.py

```diff
@@ -174,7 +191,7 @@ def _e2c(ds, supervector_dim='svd', control_member=None):
 # REFERENCE COMPARISONS
 # based on supervector approach
 # --------------------------------------------#
-def _e2r(ds, reference):
+def _e2r(ds, reference, stack=True):
     """
     Compare the ensemble mean forecast to a reference in HindcastEnsemble.
 
```

Comment:
Add description of `stack` here for developers. Can mention that it isn't used here but is placed for consistency in calling the functions.